apps/admin-gui/src/app/shared/components/dialogs/application-form-copy-items-dialog/application-form-copy-items-dialog.component.ts

```diff
@@ -61,11 +63,25 @@ export class ApplicationFormCopyItemsDialogComponent implements OnInit {
     this.loading = true;
     this.theme = this.data.theme;
     this.translateService.get('DIALOGS.APPLICATION_FORM_COPY_ITEMS.NO_GROUP_SELECTED').subscribe(
-      () => {
+      (text: string) => {
+        this.fakeGroup = {
+          id: -1,
```

Comment:
Could you please, replace the deprecated `subscribe` as well as old `TranslateService` in this file.